### ISSUE TITLE
transformation of greek/arabic transliteration in lookup results

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#344]
currently the greek and arabic morphology engines accept transliterated text in input and produce the correct results in unicode. but we don't transform the input word on the popup. We should do so.

**Comment by irina060981** [comment]
@balmas , I could work with this task, but I couldn't find an working example.
I tried to use different transliterated Greek words (but I am not a specialist at all) and all of them didn't give me any morph result.

**Comment by balmas** [comment]
oh, I'm sorry! 
Here is an example of a Greek word:

mou=sa  (greek is μοῡςα)

And an Arabic one:

bAb (arabic is باب)

**Comment by irina060981** [comment]
@balmas , I have examined the morph results for 
mou=sa (greek is μοῡςα)

and we don't have our own translitiration method.
We simply get back some results from morph tufts source like this
```
"rest": {
          "entry": {
            "uri": null,
            "dict": {
              "hdwd": {
                "lang": "grc",
                "$": "Μοῦσα"
```

so we could see transliterated in the first entry -> hdwd
but there could be more than 1 entry.

@balmas, Could we define that we should compare with first hdwd in client-adapters or homonym analysis?

I used this source to check if it could be several variants in transliteration:
https://stevemorse.org/greek/eng2grk.html

And it shows two variants for translitiration
mou=sa -> μουσα, μωυσα

That's why I believe we couldn't create any local transliteration, because we should getb those results that were used by morph adapter.

@balmas. what do you think?

**Comment by balmas** [comment]
@irina060981 , that's a good point. I think you're right that we should take it from the morph service output but I think that requires a change to both the morphsvc and the client adapter.  Since the morphsvc is doing the conversion, it should convert the word it looked up for us, and not just the results.

Right now it puts the word in the description of the target

(e.g. hasTarget": {"Description": {"about": "urn:word:mou=sa"} )

It should represent that in unicode rather than the transliteration, and then the client adapter could use that as the target word in the Homonym. 

Does that sound about right to you?

**Comment by irina060981** [comment]
Yes, I think it is the most correct way to go for Greek.
I believe that we could do something similiar for Arabic?

**Comment by balmas** [comment]
yes, it should work the same for both. I will enter the issue on the morphsvc.

**Comment by irina060981** [comment]
Ok, then we should postpone this issue before https://github.com/alpheios-project/morphsvc/issues/31 would be done

**Comment by balmas** [comment]
yes, thank you!
